css/wee.mixins.less

```diff
@@ -2425,9 +2425,9 @@
 \*------------------------------------*/
 
 .transform (...) {
-	-ms-transform: @arguments;
-	-webkit-transform: @arguments;
-	transform: @arguments;
+	-ms-transform+_: @arguments;
```

Comment:
@leigeber Not sure I understand what is being concatenated on the mixins without a `.check()` call inside. Could you explain?